02SimpleArraySum.rb

```diff
@@ -0,0 +1,28 @@
+#!/bin/ruby
```

Comment:
You can remove these codes. Just keep the function (lines 6 to 14). Do the same for the rest of the files.